### DISCUSSION TITLE
Cannot connect to Photoprism after adding Traefik

### DISCUSSION THREAD

**Discussion by xplosionmind** [photoprism/photoprism#4407] [Q&A] [answered]
Hi all, I am configuring Photoprism on a Raspberry Pi, connected to my router.

The configuration of Photoprism by itself is working well, also via the domain (`tommi.pics`). Unfortunately, while adding Traefik to the equation, I cannot reach the webapp. When visiting `https://tommi.pics/`, I get a blank page with two error messages:

1. At first, I see `Bad Gateway`
2. After a while, I see `Client sent an HTTP request to an HTTPS server.`

---

**Note**: Running `docker compose up -d` is successful, and I get no errors in the terminal.

- [This is my `docker-compose.yml` *before* adding Traefik](https://paste.yunohost.org/orikakavik.yaml); **working** (codename v0.1).
- [This is my `docker-compose.yml` *after* adding Traefik](https://paste.yunohost.org/efizoxapuf.yaml), plus [`traefik.yaml`](https://paste.yunohost.org/emoxesamiy.yaml); **not working**, see error above (codename v0.2).

I am wondering if you can help me understand if the problem I am experiencing is due to a bad config or if I have to ask for support on the Traefik forum (what should I ask? 😬)

**Comment by lastzero** [comment, marked as answer]
Sorry for the wait! The error "Client sent an HTTP request to an HTTPS server" indicates that your PhotoPrism instance has HTTPS enabled. However, if you are using an HTTPS reverse proxy, the internal communication between the service and the proxy should be unencrypted as the proxy is the HTTPS endpoint in this case (unless the proxy uses a public network to access the service).

For this, simply set `PHOTOPRISM_DISABLE_TLS` to `"true"`:

```
PHOTOPRISM_DISABLE_TLS: "true"
```

Also see the following section in our *Getting Started* guide:
- https://docs.photoprism.app/getting-started/using-https/#1-https-reverse-proxy

**Reply by xplosionmind** [reply]
Thank you so much! I have read that, but I forgot to change it in the config. If I find some time, I will add a clarification section about this in the docs Traefik page in the next few days.
